### PR TITLE
feat(bitgo): update tss hd wallet sharing

### PR DESCRIPTION
**Pull request by zahin-mohammad**
TICKET: BG-44569

Partial revert of https://github.com/BitGo/BitGoJS/pull/2081

### CODE REVIEW

File: modules/bitgo/src/v2/internal/tssUtils.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ import { BitGo } from '../../bitgo';
 import { MpcUtils } from './mpcUtils';
 import { Memo, Wallet } from '..';
 import { RequestTracer } from '../internal/util';
+import { Uint8ArrayTobase58 } from '@bitgo/account-lib/dist/src/coin/sol/utils';
```
Comment (joey-bai) [inline modules/bitgo/src/v2/internal/tssUtils.ts:28]: let's avoid using Sol's util for base58 conversion.

you can do `import bs58 from 'bs58'`

File: modules/bitgo/src/v2/internal/tssUtils.ts [hunk 2]
```diff
@@ -661,4 +662,18 @@ export class TssUtils extends MpcUtils {
...
+   *
+   * @param {String} commonKeychain
+   * @returns {string}
+   */
+  static getPublicKeyFromCommonKeychain(commonKeychain: string): string {
+    if (commonKeychain.length !== 128) {
+      throw new Error(`Invalid commonKeychain length, expected 128, got ${commonKeychain.length}`);
+    }
+    const commonPubHexStr = commonKeychain.slice(0, 64);
+    return Uint8ArrayTobase58(Buffer.from(commonPubHexStr, 'hex'));
```
Comment (joey-bai) [inline modules/bitgo/src/v2/internal/tssUtils.ts:677]: `return bs58.encode(Buffer.from(commonPubHexStr, 'hex'));`

File: modules/bitgo/src/v2/internal/tssUtils.ts (hunk 1 above)
Comment (zahin-mohammad) [reply modules/bitgo/src/v2/internal/tssUtils.ts]: Fixed

File: modules/bitgo/src/v2/internal/tssUtils.ts (hunk 2 above)
Comment (zahin-mohammad) [reply modules/bitgo/src/v2/internal/tssUtils.ts]: Fixed

**Review by joey-bai — DISMISSED** [review]

**Review by joey-bai — APPROVED** [review]

**Pull request merged**